transmart-server/grails-app/conf/application.groovy

```diff
@@ -213,7 +217,8 @@ grails { plugin { springsecurity {
     // requestmap in db
     securityConfigType = grails.plugin.springsecurity.SecurityConfigType.Requestmap
     // url to redirect after login in
-    successHandler.defaultTargetUrl = '/userLanding'
+    // just_rest branch provides alternative default via org.transmart.defaultLoginRedirect
+    successHandler.defaultTargetUrl = org.transmart.defaultLoginRedirect ?: '/userLanding'
```

Comment:
This is what the UserLandingController has the `com.recomdata.defaults.landing` variable for.